ci/prs.yml

```diff
@@ -20,11 +20,9 @@ jobs:
 
 - job: check_standard_change_label
   dependsOn:
-    - git_sha
-  variables:
-    fork_sha: $[ dependencies.git_sha.outputs['out.fork_point'] ]
-    branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
-  condition: eq(variables['Build.Reason'], 'PullRequest')
+    - check_for_release
+  condition: and(eq(variables['Build.Reason'], 'PullRequest'),
+                 eq(dependencies.check_for_release.outputs['out.is_release'], 'true'))
```

Comment:
This used to make sense, because we had two types of Standard-Change changes in this repo: changes to LATEST, and changes to infra. `infra` has moved out to its own repo, and `LATEST` has always been a proxy for releases, so now we can just run this when we have a release - we already have a single source of truth for that determination.